packages/components/src/Button/Button.js

```diff
@@ -0,0 +1,83 @@
+import { forwardRef } from 'react';
+import { createStyleSheet, alpha as convertRgba } from '@platzily-ui/styling';
+import { cx } from '@emotion/css';
+
+const useStyleSheet = createStyleSheet(
+  (theme, props) => {
+    return {
+      primaryButton: {
+        backgroundColor: props.color || theme.palette.secondary.main,
+        borderRadius: '5px',
```

Comment:
You could use the number instear, for example, `5` instead of `5px`. Lets use that approach. BTW, the border radius will change as soon as we define it in the theme.